addons/l10n_cl/views/report_invoice.xml

```diff
@@ -268,4 +268,42 @@
         </tr>
     </template>
 
+    <template id="report_invoice_document_copy" inherit_id="l10n_cl.report_invoice_document">
+        <xpath expr="//div[@id='right-elements']" position="inside">
+            <div t-if="show_information_box" name="custom_info_box" class="col-12 mt-3 p-3 border border-primary rounded">
```

Comment:
Seems a bit IA generated that ahah, don't rely too much on that ahah. IA is a tool that you can use but do not trust everything he say :smile: 
Missing a condition on the partner use don't you think ? Do we really want that for all customers ?